storybook/stories/materials/typography/stories/typography.stories.mdx

```diff
@@ -74,15 +74,37 @@ Body text is typically used for long-form writing as it works well for small tex
         ["f6", "lh4", "fw4", <p className="f6 lh4 pv3">If two pieces of the same type of metal touch <br/> in space they will permanently bond.</p>],
         ["f7", "lh3", "fw4", <p className="f7 lh3 pv3">If two pieces of the same type of metal touch <br/> in space they will permanently bond.</p>],
         ["f8", "lh5", "fw4", <p className="f8 lh5 pv3">If two pieces of the same type of metal touch <br/> in space they will permanently bond.</p>],
-        ["f9", "lh1", "fw4", <p className="f9 lh1 pv3">If two pieces of the same type of metal touch <br/> in space they will permanently bond.</p>]
+        ["f9", "lh1", "fw4", <p className="f9 lh4 pv3">If two pieces of the same type of metal touch <br/> in space they will permanently bond.</p>]
     ]}
 />
 
+### Label
```

Comment:
I would refrain from documenting this until further thoughts are put into it with the Orbit team and designers.

Reply:
It's available in ShareGate's UI Kit and I know that my designers use them and we have a couple of use cases in ShareGate Overcast. Since it's already in use, I would at least document how you should use them if you are to use them.

Orbit's documentation is more a developer's reference since the designers use Figma directly, I want to try to avoid having a developer change what the designer intended because the combination of font-size, font-weight and line-height are not documented in Orbit.

I can do another PR to further discuss Label so that it doesn't block the fix to Headlines and Body Texts.

Reply:
Fair point I don't disagree with exposing the Label combinations